backend-start.sh

```diff
@@ -94,6 +95,32 @@ except Exception as e:
     db.rollback()
 finally:
     db.close()
+
```

Comment:
is it not possible to manage this in an external.py file? might help even just for checking syntax errors and such